plugins/bnet_plugin/bnet_plugin.cpp

```diff
@@ -573,7 +573,7 @@ namespace eosio {
            _app_ios.post( [self = shared_from_this(),callback]{
               auto& control = app().get_plugin<chain_plugin>().chain();
               auto lib = control.last_irreversible_block_num();
-              auto head = control.head_block_id();
+              auto head = control.fork_db_head_block_id();
```

Comment:
Has irreversible mode been tested with the bnet plugin? I am less confident from just code analysis that the `head_block_*` to `fork_db_head_block_*` change is sufficient for the bnet_plugin as it was with the net_plugin. The bnet_plugin seems to maintain its own local head block numbers based on the accepted_block channel.